src/lib.rs

```diff
@@ -63,7 +63,9 @@ pub mod file_size_opts {
         /// The amount of zeroes to display of the decimal part is zero.
         pub decimal_zeroes: usize,
         /// Wether to use the full suffix or its abbreveation.
-        pub long_suffix: bool
+        pub long_suffix: bool,
+        /// Wether to place a space between value and units.
```

Comment:
Just realized we both misspelled "Whether". Oups.
(I believe a "wether" is a castrated sheep...)